### ISSUE TITLE
[BITV] Replace custom tooltips with native one

### ISSUE THREAD

**Issue by Pytal** [nextcloud/activity#1045]
Custom [Popper.js](https://popper.js.org/docs/v2/constructors/#createpopper) based tooltips used in [v-tooltip](https://github.com/Akryum/floating-vue/tree/v-tooltip-2#readme) and [Bootstrap](https://getbootstrap.com/docs/4.0/components/tooltips/) need to be replaced with native HTML [`title`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) attribute based tooltips

### Search queries
- [v-tooltip](https://github.com/search?q=repo%3Anextcloud%2Factivity+v-tooltip&type=code)
- [.tooltip](https://github.com/search?q=repo%3Anextcloud%2Factivity+.tooltip&type=code)
## Problem

To show a tooltip on a several components we currently use v-tooltip. An example can be found in https://github.com/nextcloud/activity/blob/master/js/script.js#L207

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to several components that will render as `title` attribute on the HTML element.

**Comment by Pytal** [comment]
@JuliaKirschenheuter opened this issue for you!
